test1.html

```diff
@@ -132,7 +132,7 @@ <h3><a href="#">Neck shoulder pain</a></h3>
     </section>
   </main>
   <footer class="page-footer">
-    <section class="footer-info">
+    <article class="footer-info">
       <ul class="info-list">
         <h3>ABOUT US</h3>
         <li class="info-item">GENERAL INTRODUCTION</li>
```

Comment:
Should not type upper-case this text.
Using CSS for transform the text to upper-case